test_vectors/test_bls.yml

```diff
@@ -0,0 +1,381 @@
+fork: tchaikovsky
+summary: Test vectors for BLS signature
+test_suite: bls
+title: BLS signature and aggregation tests
+version: 1.0
+case01_message_hash_G2_uncompressed:
+- input: {domain: '0x00', message: '0x6d657373616765'}
```

Comment:
According to the spec `message` has `bytes32` type. IMO, it would be better if length of these values was aligned with the type.